ksqldb-parser/src/test/java/io/confluent/ksql/parser/KsqlParserTest.java

```diff
@@ -599,6 +605,29 @@ public void testCreateStreamAsSelect() {
     assertThat(((AliasedRelation) query.getFrom()).getAlias().text().toUpperCase(), equalTo("ORDERS"));
   }
 
+  @Test
+  public void shouldFailIfSourcedByConnectorProvidedCXAS() {
+    for (final String source: ImmutableSet.of("STREAM", "TABLE")) {
+      // Given:
```

Comment:
unfortunately, we don't have parameterized tests from https://www.baeldung.com/parameterized-tests-junit-5 available